### PR TITLE
Enhacements

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

- [x] Fix minor corrections docs.
- [x] Add `Menu::class` widget.
- [x] Minor corrections widgets.

### CODE REVIEW

File: src/Menu.php [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+
+use function array_merge;
+use function array_values;
+use function call_user_func;
+use function count;
+use function implode;
+
+final class Menu extends Widget
+{
+    private string $activeCssClass = 'is-active';
```
Comment (samdark) [inline src/Menu.php:19]: Doesn't seem to be used anywhere...

File: src/Menu.php [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+    private bool $activateItems = true;
+    private bool $activateParents = false;
+    private string $brand = '';
```
Comment (samdark) [inline src/Menu.php:22]: What's the brand in the menu?

File: src/Widget.php [hunk 3]
```diff
@@ -64,10 +74,14 @@ protected function getId(): ?string
      *
      * @return array
      */
-    protected function addOptions(array $options, string $valueDefault): array
+    protected function addOptions(array $options, string $valueDefault, array $params = []): array
```
Comment (samdark) [inline src/Widget.php:77]: That looks a bit weird... why do we have two sources for the same `$options`?

**Review by samdark — CHANGES_REQUESTED** [review]

File: src/Menu.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Menu.php:19]: Fixed done.

File: src/Menu.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Menu.php:22]: Title menu:

image

File: src/Widget.php (hunk 3 above)
Comment (terabytesoftw) [reply src/Widget.php:77]: Fixed.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**